app/com/linkedin/drelephant/DrElephant.java

```diff
@@ -23,20 +23,20 @@
  * The main class which starts Dr. Elephant
  */
 public class DrElephant extends Thread {
-  private ElephantRunner _elephant;
+    private ElephantRunner _elephant;
```

Comment:
dont change indentation of a public OS project. Correct it to reflect only your change